packages/kolibri-tools/lib/logging.js

```diff
@@ -4,38 +4,42 @@
  * @module logging
  */
 
-var colors = require('colors');
+const colors = require('colors');
 
-var logging = {
+const logging = {
   /**
    * Log a message in a particular colour.
    * N.B. This is only for command line output, not in the browser.
    * @param {string} msg - The msg to log.
    * @param {Object} color - A function that maps the colour of the string for output.
    */
   prefix: 'Kolibri: ',
-  log(msg, color, prefixed = true) {
-    /* eslint-disable no-console */
-    if (color && prefixed) {
-      console.log(color(this.prefix + msg));
-    } else if (color) {
-      console.log(color(msg));
-    } else {
-      console.log(msg);
-    }
-    /* eslint-enable no-console */
+  write(level, messages, color, prefix = this.prefix) {
+    messages = prefix ? messages.map(msg => this.prefix + msg) : messages;
```

Comment:
I don't think this is quite intended behaviour - as passing multiple args to a console function prints it all on the same line, I think it only needs one prefix? So perhaps it would be better as:
`messages = prefix ? [prefix].concat(messages) : messages;`

Reply:
Yep, whoops.

Reply:
(which probably also means we can get rid of the space at the end of the prefixes!)

Reply:
That's better.
![logger](https://user-images.githubusercontent.com/819838/57030335-2b5c2a80-6bf9-11e9-9a58-93020bbd0e27.png)